src/ui/screens/AddAccountScreen/AddAccountScreen.test.tsx

```diff
@@ -0,0 +1,26 @@
+import {NavigationProp} from '@react-navigation/native';
+import {AppStackParamList} from '@ui/navigation/navigation';
+import React from 'react';
+import {render, waitFor, fireEvent} from 'src/testUtils';
+import {AddAccountScreen} from './AddAccountScreen';
+
+const navigation = {
+  goBack: () => jest.fn,
+} as unknown as NavigationProp<AppStackParamList>;
+
+test('render the loading view when rendered with no data', async () => {
+  const navigationSpy = jest.spyOn(navigation, 'goBack');
+  const {getByPlaceholderText, getByText} = render(<AddAccountScreen navigation={navigation} />);
+  await waitFor(() => {
+    expect(getByText('Add external account')).toBeTruthy();
+    const inputAddress = getByPlaceholderText('👉 Paste address here, e.g. 167r...14h');
+    expect(inputAddress).toBeTruthy();
+    fireEvent.changeText(inputAddress, '14yx4vPAACZRhoDQm1dyvXD3QdRQyCRRCe5tj1zPomhhS29a');
+    expect(getByText(/Cancel/i)).toBeDefined();
+    fireEvent.press(getByText(/Cancel/i));
```

Comment:
Trying to get the instance of a button by `getByText` can cause problem if we have similar texts rendered in the screen which are not buttons.

If we need a test instance of a button, `getAllByA11yRole('button')` and `getRole('button')` should be used.

I found some PRs already got merged like that but this is for future reference and also we can refactor when touching those components and test cases.